### ISSUE TITLE
Add support for Qdrant

### ISSUE THREAD

**Issue by NivekT** [hegelai/prompttools#47]
### 🚀 The feature

[Qdrant](https://github.com/qdrant/qdrant) is a popular vector similarity search engine and vector database.

### Motivation, pitch

We can integrate with its APIs to allow experimentation on various configurations and look at its performance.

### Additional context

You can have a look at the Chroma or Weaviate for inspiration. If you would like to work on this, comment and let us know! We will be more than happy to support you.

**Comment by NivekT** [comment]
Quote from @rafidude from #52:
```
We are using Qdrant and found its performance much better than other Vector DBs.

Please see the benchmarks at https://qdrant.tech/benchmarks/.
```

**Comment by kacperlukawski** [comment]
@NivekT I'm already working on the integration.

**Comment by NivekT** [comment]
Wonderful!

**Comment by kacperlukawski** [comment]
The PR is open, but still in progress: #54

**Comment by NivekT** [comment]
Closed by #54 
Thank you @kacperlukawski !
